### ISSUE TITLE
You blocked me on discord.js, and I don't know why

### ISSUE THREAD

**Issue by LoganDark** [abalabahaha/eris#79]
I got blocked on discord.js repo by you, and I don't know why. You limited the conversation to collaborators as soon as I tried to post that it worked. What happened and why did you do this?

**Comment by brussell98** [comment]
This is not related to eris, go away.

**Comment by LoganDark** [comment]
Sorry. Private messaging is dead, that's all. I won't do it again.

**Comment by abalabahaha** [comment]
The issue was getting pretty out of hand, and your questions could have been easily answered through Google or the new docs. The people active in the channel collectively decided to lock the issue.

Also, in case you didn't notice, this isn't the discord.js repo.

**Comment by amishshah** [comment]
smh \* 10000
